src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -94,9 +94,9 @@ public void createFeedbackSessions(Collection<FeedbackSessionAttributes> feedbac
             Date standardStart = TimeHelper.convertToUserTimeZone(startCal, fs.timeZone - zone).getTime();
             Date standardEnd = TimeHelper.convertToUserTimeZone(endCal, fs.timeZone - zone).getTime();
             
-            boolean isStartTimeWithinRange = TimeHelper.isStartTimeWithinRange(standardStart, standardEnd, fs.startTime);
-            boolean isEndTimeWithinRange = TimeHelper.isEndTimeWithinRange(standardStart, standardEnd, fs.endTime);
-            
+            boolean isStartTimeWithinRange = TimeHelper.isTimeWithinPeriod(standardStart, standardEnd, fs.startTime, true, false);
+            boolean isEndTimeWithinRange = TimeHelper.isTimeWithinPeriod(standardStart, standardEnd, fs.endTime, false, true);
```

Comment:
The code has improved a lot since the beginning. I was wondering if we can replace magic values true/false with something like `TimeHelper.START_TIME_INCLUSIVE` etc. That will increase the readability.